src/ResourceManager/RedisCache/Commands.RedisCache/Commands/ResetAzureRedisCache.cs

```diff
@@ -0,0 +1,70 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.RedisCache
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.RedisCache.Properties;
+    using Microsoft.Azure.Management.Redis.Models;
+    using RebootTypeStrings = Microsoft.Azure.Management.Redis.Models.RebootType;
+
+    [Cmdlet(VerbsCommon.Reset, "AzureRmRedisCache"), OutputType(typeof(bool))]
+    public class ResetAzureRedisCache : RedisCacheCmdletBase
```

Comment:
This is a property of the Cmdlet attribute.  It indicates that the cmdlet supports SHouldProcess.  Any cmdlet that calls Cmdlet.ShouldProcess needs to have this attribute.